onlineapi/src/test/java/com/codingblocks/onlineapi/api/OnlineJsonApiAuthenticatedTest.kt

```diff
@@ -47,4 +49,24 @@ class OnlineJsonApiAuthenticatedTest {
         assertNotNull(runAttempt)
         assertTrue(runAttempt!!.id == "80179")
     }
+
+    @Test
+    fun `GET wishlist`() {
+        //89 Course is C++ Fundamentals Course
+        val wishlistAdd = runBlocking { CBOnlineLib.onlineV2JsonApi.addWishlist(Wishlist(Course("89"))).body() }
+        assertNotNull(wishlistAdd)
+        assertTrue(wishlistAdd?.course?.id=="89")
+
+        val wishlistGet = runBlocking { CBOnlineLib.onlineV2JsonApi.getWishlist().body()?.get() }
+        assertNotNull(wishlistGet)
+        assertTrue(wishlistGet!!.isNotEmpty())
```

Comment:
Here you can test that if the list is not empty then a course with id 89 should exist in the list